### PR TITLE
:sparkles: change name to rakun

### PR DESCRIPTION
**Pull request by guillaumecrespel**
image

- [x] logo
- [x]  text + link
- [ ] work-with-us
- [ ] CNAME
- [x] markdown

image

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+username: rakun
 user_description: Bicycle rights irony actually neutra typewriter lyft. Man bun taxidermy put a bird on it, umami yr ramps pop-up ugh bushwick chia lo-fi. Occupy bitters pour-over snackwave you probably haven't heard of them small batch. Cornhole mustache man bun letterpress echo park VHS.
 user_title: à deux, c’est mieux !
 
 company: 
-  twitter: alakarteio
-  github: alakarteio
+  twitter: RakunTeam
+  github: rakunteam
   linkedin: alakarteio
```
Comment (guillaumecrespel) [inline _config.yml]: it's not change on linkedin :(

**Comment by guillaumecrespel** [comment]
I've not change the CNAME to.

File: _config.yml (hunk 1 above)
Comment (fabienjuif) [reply _config.yml]: I just fixed that

**Comment by fabienjuif** [comment]
I just change DNS to point at github IPs.
I changed the CNAME but we have to wait them to issue a SSL certificat (24H)

File: _config.yml [hunk 2]
```diff
@@ -1,19 +1,19 @@
 # Site settings
-title: alakarteio
+title: rakun - team
```
Comment (fabienjuif) [inline _config.yml]:
```suggestion
title: rakun
```

**Review by fabienjuif — APPROVED** [review]

**Pull request merged**